### ISSUE TITLE
Exclude for BlockPlacementRule doesn't work

### ISSUE THREAD

**Issue by krzys-h** [colobot/colobot-lint#17]
I tried to exclude a fragment of the code from BlockPlacementRule as a workaround to #16 but it didn't work.
The code was something like this:
```C++
void CEngine::Something()
{
    auto sth = []()
    {
        return 5;
    // @colobot-lint-exclude BlockPlacementRule
    }();
    // @end-colobot-lint-exclude
}
```
I also tried this but it didn't work either:
```C++
void CEngine::Something()
{
    // @colobot-lint-exclude BlockPlacementRule
    auto sth = []()
    {
        return 5;
    }();
    // @end-colobot-lint-exclude
}
```

**Comment by krzys-h** [comment]
Whoops, I was looking at a wrong error all this time.
